### PR TITLE
Docs SEO improvements

### LINKED ISSUE #401: Improve docs SEO
This is HubSpot's SEO review of the install page: 
screen shot 2016-06-15 at 12 35 29 pm

@jess-edwards

### PR DESCRIPTION
**Pull request by jseldess**
Adding alt-text to images and meta descriptions to pages. The Jekyll them uses the summary field in page front-matter for meta descriptions. Fixes #401

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/403)

### CODE REVIEW

File: build-a-test-app.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Build a Test App
+summary: Follow this tutorial to quickly learn how to create a CockroachDB database and connect to it from a client applicaiton.
```
Comment (jess-edwards) [inline build-a-test-app.md]: application.

File: cloud-deployment.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Cloud Deployment
+summary: Deploy CockroachDB on Google Cloud Engine or AWS.
```
Comment (jess-edwards) [inline cloud-deployment.md]: Learn how to deploy...

Comment (jseldess) [inline cloud-deployment.md]: Fixed.

File: build-a-test-app.md (hunk 1 above)
Comment (jseldess) [inline build-a-test-app.md]: Good catch. Fixed.

File: index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: CockroachDB Docs
+summary: CockroachDB is a distributed SQL database built on a transactional and strongly-consistent key-value store.
```
Comment (jess-edwards) [inline index.md]: "Read the documentation for CockroachDB, a distributed SQL database built on.... 

[This is to distinguish this meta description from the main homepage's meta description.]

File: install-cockroachdb.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Install CockroachDB
+summary: Install CockroachDB on Mac, Linux, or Windows.
```
Comment (jess-edwards) [inline install-cockroachdb.md]: Simple instructions for installing CockroachDB on Mac, Linux, or Windows.

File: tech-talks.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Tech Talks
+summary: Watch recordings of CockroachDB founders and engineers giving high-level and detailed talks about the database.
```
Comment (jess-edwards) [inline tech-talks.md]: detailed tech talks about the database.

**Comment by jess-edwards** [comment]
@jseldess LGTM after you address my last few comments. 

I wish I could make updates to the WP site like this...

File: index.md (hunk 3 above)
Comment (jseldess) [inline index.md]: Done with next commit.

File: install-cockroachdb.md (hunk 4 above)
Comment (jseldess) [inline install-cockroachdb.md]: Done with next commit.

File: tech-talks.md (hunk 5 above)
Comment (jseldess) [inline tech-talks.md]: Done with next commit.

**Pull request merged**